src/lib.rs

```diff
@@ -687,63 +665,67 @@ impl Logger {
         if let Some(v) = self.verbosity {
             self.level = match v + self.offset {
-                0 => LogLevel::Error,  
-                1 => LogLevel::Warn,  
-                2 => LogLevel::Info,  
-                3 => LogLevel::Debug, 
-                _ => LogLevel::Trace, 
+                0 => log::Level::Error,
+                1 => log::Level::Warn,
+                2 => log::Level::Info,
+                3 => log::Level::Debug,
+                _ => log::Level::Trace,
             };
         }
-        log::set_logger(|max_level| {
-            max_level.set(self.level.to_log_level_filter());
-            Box::new(self)
-        })
+        log::set_max_level(self.level.to_level_filter());
+        log::set_boxed_logger(Box::new(self))
     }
 
     /// Gets the color to use for the log statement's tag based on level.
-    fn select_color(&self, l: &LogLevel) -> Colour {
+    fn select_color(&self, l: &log::Level) -> Colour {
         match *l {
-            LogLevel::Error => self.error.color,
-            LogLevel::Warn => self.warn.color,
-            LogLevel::Info => self.info.color,
-            LogLevel::Debug => self.debug.color,
-            LogLevel::Trace => self.trace.color,
+            log::Level::Error => self.error.color,
+            log::Level::Warn => self.warn.color,
+            log::Level::Info => self.info.color,
+            log::Level::Debug => self.debug.color,
+            log::Level::Trace => self.trace.color,
         }
     }
 
     /// Gets the output stream to use for the level.
-    fn select_output(&self, l: &LogLevel) -> Output {
+    fn select_output(&self, l: &log::Level) -> Output {
         match *l {
-            LogLevel::Error => self.error.output,
-            LogLevel::Warn => self.warn.output,
-            LogLevel::Info => self.info.output,
-            LogLevel::Debug => self.debug.output,
-            LogLevel::Trace => self.trace.output,
+            log::Level::Error => self.error.output,
+            log::Level::Warn => self.warn.output,
+            log::Level::Info => self.info.output,
+            log::Level::Debug => self.debug.output,
+            log::Level::Trace => self.trace.output,
         }
     }
 
-    /// Creates the tag portion of the log statement based on the configuration. 
+    /// Creates the tag portion of the log statement based on the configuration.
     ///
     /// The tag portion is the of the log statement is the text to the left of the separator, while
     /// the text to the right of the separator is the message.
-    fn create_tag(&self, record: &LogRecord) -> String {
+    fn create_tag(&self, record: &log::Record) -> String {
         let level = record.level();
         let level_text = if self.include_level {
             level.to_string()
         } else {
             String::new()
         };
+
         let module_path_text = if self.include_module_path {
+            let pth = record.module_path().unwrap_or("missing");
```

Comment:
note: this should probably be `"<unknown>"` - that's what it was in the forward breaking period, but not sure it's needed.
I haven't seen this trigger yet.